docs/References.bib

```diff
@@ -354,6 +354,24 @@ @article{Teukolsky2015ega
   SLACcitation   = "%%CITATION = ARXIV:1510.01190;%%"
 }
 
+@article{White2015omx,
+  author         = "White, Christopher J. and Stone, James M. and Gammie,
+                    Charles F.",
+  title          = "{An Extension of the Athena++ Code Framework for GRMHD
+                    Based on Advanced Riemann Solvers and Staggered-Mesh
+                    Constrained Transport}",
+  journal        = "Astrophys. J. Suppl.",
+  volume         = "225",
+  year           = "2016",
+  number         = "2",
+  pages          = "22",
+  doi            = "10.3847/0067-0049/225/2/22",
```

Comment:
Add url field with this doi please (see the guidelines linked at the top of this file)